app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -71,6 +71,36 @@ public class MainActivity extends MarkorBaseActivity implements GsFileBrowserFra
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+
+        if (_appSettings.getAppThemeName().contains("default")) {
+            setTheme(R.style.AppTheme_Unified);
+        }
+
+        if (_appSettings.getAppAccentName().contains("black_white")) {
```

Comment:
When it's done this way, next time people will complain that they can't customize colors. While it's more possibilities, it's still limited to a small amount of hardcoded colors this way.